test/src/tests-store.ts

```diff
@@ -961,6 +961,78 @@ describe('Store', () => {
         });
     });
 
+    describe('#syncPersistenceData', () => {
+        it('should sync store with persistence values', () => {
+            const persistenceValue = JSON.stringify({
+                testMPID: {
+                    lst: 12345,
+                    fst: 54321,
+                    con: sampleConsentStateFromPersistence.con,
+                },
+            });
+
+            localStorage.setItem(workspaceCookieName, persistenceValue);
+
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            store.syncPersistenceData();
+
+            const fromPersistence = window.mParticle
+                .getInstance()
+                ._Persistence.getPersistence();
+
+            expect(fromPersistence[testMPID]).to.be.ok;
+            expect(fromPersistence[testMPID].lst).to.be.ok;
+            expect(fromPersistence[testMPID].lst).to.equal(12345);
+
+            expect(fromPersistence[testMPID].fst).to.be.ok;
+            expect(fromPersistence[testMPID].fst).to.equal(54321);
+
+            expect(fromPersistence[testMPID].con).to.be.ok;
+            expect(fromPersistence[testMPID].con).to.deep.equal(
+                sampleConsentStateFromPersistence.con
+            );
```

Comment:
Since this test is syncing the `store with persistence values`, shouldn't we be testing the store's persistence data?  

I think that if you didn't call `store.syncPersistenceData()`, these tests would still pass because you are setting items on localStorage manually, and then calling `getPersistence`, which goes to localStorage, and doesn't does the `store` right?

Reply:
My fault. I made the wrong assertions.